pkg/new-ui/v1beta1/frontend/src/app/pages/experiment-creation/hyper-parameters/parameter/parameter.component.html

```diff
@@ -0,0 +1,70 @@
+<div class="params-row">
```

Comment:
Do you need this parameter  since you have the common one: https://github.com/kubeflow/katib/tree/1b48a137ac24428d83bd411bc1ce4d3f4724d1c6/pkg/new-ui/v1beta1/frontend/src/app/shared/params-list/parameter ?

Reply:
No, it can be completely removed. The shared one was based, and is identical, to this one.
The code was already using the shared one so this is only something I forgot to remove.

pushed another commit for this